demos/continuous_batching/accuracy/README.md

```diff
@@ -107,7 +103,33 @@ openai_compatible (model_version=OpenGVLab/InternVL2_5-8B,max_retries=1), gen_kw
 ```
 
 
+## Running the tests for agentic models with function calls
+
+Use [Berkeley function call leaderboard ](https://github.com/ShishirPatil/gorilla/tree/main/berkeley-function-call-leaderboard)
+
+
+```bash
+git clone https://github.com/ShishirPatil/gorilla
+cd gorilla/berkeley-function-call-leaderboard
+git checkout ac37049f00022af54cc44b6aa0cad4402c22d1a0
+git apply gorila.patch
```

Comment:
I get: 
error: can't open patch 'berkeley-function-call-leaderboard/gorila.patch': No such file or directory

Reply:
@dtrawins I can confirm the issue. Following the instruction at this point we are inside gorilla repository, but the patch is in model_server/demos/continuous_batching/accuracy, so this line should be modified